packages/client/src/Loc.ts

```diff
@@ -429,6 +429,7 @@ export abstract class LocRequestState extends State {
         return {
             ...request,
             requesterAddress: request.requesterAddress || undefined,
+            requesterLocId: request.requesterIdentityLoc ? new UUID(request.requesterIdentityLoc) : undefined,
```

Comment:
I am not sure we are enforcing backend-side that only LOs can actually create logion LOCs.

Reply:
As discussed, this check is done by the node runtime. On the backend, we still check that the request is submitted by either the LO or the requester. For transactions/collections we also check there is a valid identity LOC. Those checks are sufficient IMHO.